rfc9069.xml

```diff
@@ -86,18 +86,10 @@
 
                 BMP <xref target="RFC7854" format="default"/> does not define a method to send
                 the BGP instance Loc-RIB. 
-<!-- [rfced] Example clarity: The "for example" text is a sentence fragment, and it is unclear how it relates to the text that precedes it.  Please clarify.
-
-Original:
-   It does define in section 8.2 of [RFC7854] locally
-   originated routes, but these routes are defined as the routes
-   originated into BGP.  For example, as defined by Section 9.4 of
-   [RFC4271].
--->
 
  It does define locally originated routes in <xref target="RFC7854" sectionFormat="of" section="8.2"/>,
-                but these routes are defined as the routes that originated into BGP. For example, as
-                described by <xref target="RFC4271" sectionFormat="of" section="9.4"/>. Loc-RIB
+                but these routes are defined as the routes that originated (e.g., 
+                <xref target="RFC4271" sectionFormat="of" section="9.4"/>) into BGP. Loc-RIB
```

Comment:
@sginoza , so I should avoid the eg in favor of spelling it out?

Reply:
@TimEvens  Hi Tim, 
Sorry I wasn't clear.  I  suggest moving "(e.g., Section 9.4 of [RFC4271])" to the end of the sentence for readability.  So, we'd have the following.  If you prefer your update, that's ok - this is minor.

   It does define locally originated routes in
   Section 8.2 of [RFC7854], but these routes are defined as the routes
   that originated into BGP (e.g., Section 9.4 of [RFC4271]).